src/lang/german.txt

```diff
@@ -1737,6 +1737,10 @@ STR_CONFIG_SETTING_NOSERVICE                                    :Wartung deaktiv
 STR_CONFIG_SETTING_NOSERVICE_HELPTEXT                           :Schicke Fahrzeuge nicht zur Wartung, wenn Pannen ausgeschaltet sind
 STR_CONFIG_SETTING_WAGONSPEEDLIMITS                             :Berücksichtige Waggonhöchstgeschwindigkeit: {STRING}
 STR_CONFIG_SETTING_WAGONSPEEDLIMITS_HELPTEXT                    :Begrenze die Höchstgeschwindigkeit eines Zuges durch die jeweiligen Höchstgeschwindigkeiten der mitgeführten Waggons
+STR_CONFIG_SETTING_TRAIN_SPEED_ADAPTION                         :Zuggeschwindigkeitsanpassung aktivieren: {STRING2}
+STR_CONFIG_SETTING_TRAIN_SPEED_ADAPTION_HELPTEXT                :Wenn aktiviert, passen schnellere Züge hinter langsameren Zügen ihre Geschwindigkeit an.
+STR_CONFIG_SETTING_SLOW_ROAD_VEHICLES_IN_CURVES                 :Straßenfahrzeuge werden in Kurven langsamer: {STRING2}
```

Comment:
When translating to other languages, always use just `{STRING}` in place of `{STRING2}` except in `english.txt`.

Reply:
Gotcha